src/extension.ts

```diff
@@ -6,15 +6,15 @@ import { ExtensionContext } from 'vscode';
 
 export async function activate(context: ExtensionContext) {
     initLogger(context);
-    const logger = getLogger().getChildLogger({label: "activate"});
+    const logger = getLogger().getChildLogger({ label: "activate" });
 
     startBasctlServer();
-    
+
     ActionsController.loadActions();
 
     ActionsController.performScheduledActions();
 
-    ActionsController.performActionsFromParams();
+    await ActionsController.performActionsFromParams();
```

Comment:
is it intended - activation time is changed ?

Reply:
done